pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/barnepensjon/BarnepensjonInnvilgelse.kt

```diff
@@ -8,19 +8,27 @@ import no.nav.pensjon.brev.template.dsl.text
 import no.nav.pensjon.brevbaker.api.model.LetterMetadata
 import no.nav.pensjon.etterlatte.EtterlatteBrevKode
 import no.nav.pensjon.etterlatte.EtterlatteTemplate
+import no.nav.pensjon.etterlatte.maler.Avdoed
 import no.nav.pensjon.etterlatte.maler.AvdoedSelectors.doedsdato
 import no.nav.pensjon.etterlatte.maler.AvdoedSelectors.navn
-import no.nav.pensjon.etterlatte.maler.BarnepensjonInnvilgelseDTO
-import no.nav.pensjon.etterlatte.maler.BarnepensjonInnvilgelseDTOSelectors.avdoed
-import no.nav.pensjon.etterlatte.maler.BarnepensjonInnvilgelseDTOSelectors.utbetalingsinfo
+import no.nav.pensjon.etterlatte.maler.Avkortingsinfo
+import no.nav.pensjon.etterlatte.maler.Utbetalingsinfo
 import no.nav.pensjon.etterlatte.maler.UtbetalingsinfoSelectors.antallBarn
 import no.nav.pensjon.etterlatte.maler.UtbetalingsinfoSelectors.beloep
 import no.nav.pensjon.etterlatte.maler.UtbetalingsinfoSelectors.beregningsperioder
 import no.nav.pensjon.etterlatte.maler.UtbetalingsinfoSelectors.soeskenjustering
 import no.nav.pensjon.etterlatte.maler.UtbetalingsinfoSelectors.virkningsdato
+import no.nav.pensjon.etterlatte.maler.barnepensjon.BarnepensjonInnvilgelseDTOSelectors.avdoed
+import no.nav.pensjon.etterlatte.maler.barnepensjon.BarnepensjonInnvilgelseDTOSelectors.utbetalingsinfo
 import no.nav.pensjon.etterlatte.maler.fraser.barnepensjon.Barnepensjon
 import no.nav.pensjon.etterlatte.maler.fraser.common.Vedtak
 
+data class BarnepensjonInnvilgelseDTO(
+    val utbetalingsinfo: Utbetalingsinfo,
+    val avkortingsinfo: Avkortingsinfo? = null,
+    val avdoed: Avdoed
+)
+
```

Comment:
Vet ikke om jeg er helt fan av å dra DTOer inn i Brevmalen 🤔 

Men samtidig synes jeg det fort blir mye rot med _alt_ i en fil (`EtterlatteBrev.kt`)... Kanskje heller opprtte nye filer for DTOene og la det som er felles forbli i EtterlatteBrev? Bare en tanke

Reply:
Skal røske litt meir i det her når eg tar neste runde på omgjøring av farskap snart, så ser på det da. Trur eg liker tanken på eiga fil per DTO 🤔